.semaphore/semaphore.yml

```diff
@@ -19,7 +19,9 @@ blocks:
         commands:
           - checkout --use-cache
           - cache restore
-          - bundle install --deployment -j 4 --path vendor/bundle
+          - bundle config set deployment 'true'
+          - bundle config set path 'vendor/bundle'
```

Comment:
shall we update the renuo-cli template as well?

Reply:
https://github.com/renuo/renuo-cli/pull/86